azavea/pfb-network-connectivity
# Load prototype styles and navbar

## Overview

- Removes all the old styles and replaces them with the styles from the prototype.
- Does a rough first pass at modifying the navbar to show admin links on admin views and public links otherwise.
- Loads the html for the home view but doesn't implement any of the behavior.

### Demo

![screen shot 2017-04-17 at 12 55 51-fullpage](https://cloud.githubusercontent.com/assets/6598836/25096198/3f0cd40c-236d-11e7-8486-48d102a1b0b7.png)
_Looks a lot like the prototype. Surprise!_

### Notes

- Removing all the old styles broke the existing admin site.  It's still functional, including the filters, but it doesn't look good.

![screen shot 2017-04-17 at 13 00 41](https://cloud.githubusercontent.com/assets/6598836/25096376/ebe7beee-236d-11e7-84c1-a5cf9bf6d4c4.png)

I tried briefly to fix it--i.e. find the specific styles from the old site that are missing and pull them in--but didn't succeed and figured it should go into another card rather than delaying this blocker.

- I've been having some staleness issues that I haven't managed to track down--e.g. switching to this branch from develop leaves me with not-fully-broken admin pages, but rebuilding and restarting the container clears them out.  Again, there's something to track down in the build setup but it seems like not a big enough issue to hold everything up.

## Testing Instructions

Rebuild and start the angular container and view the site.  Going to http://localhost:9301/ should now show the public site home page rather than forwarding you to login.
Going to http://localhost:9301/#/login/ and logging in should get you to the admin pages (sans styling) and admin header.

Resolves #291.
